tests/integration/views/account/test_transfer_surveys.py

```diff
@@ -4,9 +4,11 @@
 import requests_mock
```

Comment:
These tests had a considerable amount of redundancy, I have removed a lot, but there might be more